### PR TITLE
Add guide to career fairs

### PR DESCRIPTION
**Pull request by lukemorton**

### CODE REVIEW

File: guides/hiring/career_fairs.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Made Tech guide to career fairs
+
+## What does a careers fair look like?
+
+There are many stalls setup as a circuit around in a big hall. Students pass by the stalls asking questions about job opportunities and internships.
+
+Most stalls have two representatives standing to the side or in front of them. Standing behind the desk seemed to put too much of a barrier between students and representatives. Stalls typically had table clothes of the companies colors over them, some with logos. All stalls had big banners behind them with bullet point info about the companies and job opportunities at eye level. A lot of stalls had sweets or chocolate on them along with leaflets and swag ranging from totes, USB sticks, popcorn, pens, paper pads, rubix cubes. Some stalls had challenges like solving a rubix cube with a leaderboard. Some had examples of apps they'd built on laptops. A few stalls had additional podiums they brought a long. Our google glasses were another example of things on tables.
```
Comment (zuren) [inline guides/hiring/career_fairs.md]: Google cardboards*

File: guides/hiring/career_fairs.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+## The patter(n)
+
+At the start we weren't sure on how to begin conversations but throughout the day we built up a fairly consistent conversation structure that worked for us:
+
+ - Ask whether the student is studying compsci or has touched any code?
+ - Describe the fact we primariy build software, from anything like e-commerce stores, to complex warehousing APIs and even back office tools.
+ - Highlight the fact we go into other businesses to help their teams deliver better software faster. Several times we described moving organisations from delivering software every 3 months to every day.
+ - Go into detail about our code dojos, hack days, retreat, blogging and advise them to read our blog and github to see how we work.
+ - Provide google glasses to take a look at the office.
```
Comment (zuren) [inline guides/hiring/career_fairs.md]: cardboard*

File: guides/hiring/career_fairs.md (hunk 1 above)
Comment (RyanMacG) [inline guides/hiring/career_fairs.md]: Google Cardboard not glasses

File: guides/hiring/career_fairs.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+ - Try and take details or at least give a leaflet to every person who approaches.
+ - Also invite people to our office, to code dojos or whatever else if you fancy.
+ 
+## Things to try next time
+
+ - Name badges with Made Tech logo on them
+ - Sweets are very popular like cadburys celebrations
```
Comment (zuren) [inline guides/hiring/career_fairs.md]: Something vegan too? Might sound small but would make a difference to me.

File: guides/hiring/career_fairs.md (hunk 2 above)
Comment (RyanMacG) [inline guides/hiring/career_fairs.md]: Same as above

File: guides/hiring/career_fairs.md (hunk 3 above)
Comment (lukemorton) [inline guides/hiring/career_fairs.md]: Like it.

**Comment by chrisblackburn** [comment]
Can we capitalise all the company names appropriately? #pethate

File: guides/hiring/career_fairs.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+## The patter(n)
+
+At the start we weren't sure on how to begin conversations but throughout the day we built up a fairly consistent conversation structure that worked for us:
+
+ - Ask whether the student is studying compsci or has touched any code?
+ - Describe the fact we primariy build software, from anything like e-commerce stores, to complex warehousing APIs and even back office tools.
+ - Highlight the fact we go into other businesses to help their teams deliver better software faster. Several times we described moving organisations from delivering software every 3 months to every day.
+ - Go into detail about our code dojos, hack days, retreat, blogging and advise them to read our blog and github to see how we work.
+ - Provide Google Cardboard to take a look at the office.
+ - Try and take details or at least give a leaflet to every person who approaches.
```
Comment (chrisblackburn) [inline guides/hiring/career_fairs.md]: Try to

**Pull request merged**